tests/unit/AddStaffNotationDialog.spec.ts

```diff
@@ -360,8 +360,7 @@ describe('AddStaffNotationDialog', () => {
       {
         propsData: {
           dialog: true,
-          displayName: 'Court Order',
-          name: 'courtOrder'
```

Comment:
You can't remove `name`. First, it's a required prop. Second, it's used to determine the type of filing.

Reply:
Totally my bad. Should not have been committed like this. Thanks for pointing out.